tests/testing_common.py

```diff
@@ -263,6 +274,32 @@ def check_config_json(self, tmp_dirname, model):
         if hasattr(model, "config"):  # custom models don't have a config attribute
             assert config["base_model_name_or_path"] == model.config.to_dict()["_name_or_path"]
 
+    def perturb_trainable_token_weights_if_used(self, model, config_kwargs, adapter_name="default", weight=1.0):
```

Comment:
```suggestion
    def perturb_trainable_token_weights_if_used(self, model, config_kwargs, adapter_name="default", scale=1.0):
```

Maybe more precise name?